common/notifications.cpp

```diff
@@ -980,4 +979,61 @@ namespace rs2
 
         pinned = true;
     }
+
+    metadata_warning_model::metadata_warning_model()
+        : notification_model()
+    {
+        enable_expand = false;
+        enable_dismiss = true;
+        pinned = true;
+        message = "Frame Metadata is a device feature allowing\n"
+                  "software synchronization between different\n"
+                  "camera streams.\n"
+                  "It must be explicitly enabled on Windows OS\n";
+    }
+
+    void metadata_warning_model::set_color_scheme(float t) const
+    {
+        notification_model::set_color_scheme(t);
+        ImGui::PopStyleColor(1);
+        auto c = alpha(sensor_bg, 1 - t);
+        ImGui::PushStyleColor(ImGuiCol_WindowBg, c);
+    }
+
+    void metadata_warning_model::draw_content(ux_window& win, int x, int y, float t, std::string& error_message)
+    {
+        ImGui::SetCursorScreenPos({ float(x + 9), float(y + 4) });
+
+        ImVec4 shadow{ 1.f, 1.f, 1.f, 0.1f };
+        ImGui::GetWindowDrawList()->AddRectFilled({ float(x), float(y) },
+        { float(x + width), float(y + 25) }, ImColor(shadow));
+
+        ImGui::Text("Frame Metadata Disabled");
+
+        ImGui::SetCursorScreenPos({ float(x + 5), float(y + 27) });
+
+        ImGui::PushStyleColor(ImGuiCol_Text, light_grey);
+        draw_text(get_title().c_str(), x, y, height - 50);
+        ImGui::PopStyleColor();
+
+        ImGui::SetCursorScreenPos({ float(x + 5), float(y + height - 25) });
+
+        auto sat = 1.f + sin(duration_cast<milliseconds>(system_clock::now() - created_time).count() / 700.f) * 0.1f;
+        ImGui::PushStyleColor(ImGuiCol_Button, saturate(sensor_header_light_blue, sat));
+        ImGui::PushStyleColor(ImGuiCol_ButtonHovered, saturate(sensor_header_light_blue, 1.5f));
+        std::string button_name = to_string() << "Enable" << "##enable_metadata" << index;
+
+        const auto bar_width = width - 115;
+        if (ImGui::Button(button_name.c_str(), { float(bar_width), 20.f }))
+        {
+            metadata_helper::instance().enable_metadata();
+            dismiss(false);
+        }
+        ImGui::PopStyleColor(2);
+
+        if (ImGui::IsItemHovered())
+        {
+            ImGui::SetTooltip("%s", "You will be prompted to enabled metadata on the device");
```

Comment:
"to enable" (there's an extra d in there)

May I suggest:

"Enables metadata on connected devices (you may be prompted for administrator privileges)"